src/eventQueue.ts

```diff
@@ -56,14 +59,21 @@ eventQueue.on(QueueEvents.QueueUpdated, function (this: EventQueue) {
 /*
  * Add a new event to the queue
  */
-eventQueue.on(QueueEvents.NewEvent, function (this: EventQueue, event: ContractEvent) {
-  this.queue.push(event);
-  verbose('Event added to the queue:', event?.signature || ContractEventsSignatures.UknownEvent);
-  this.emit(QueueEvents.QueueUpdated);
-});
+eventQueue.on(
+  QueueEvents.NewEvent,
+  function (this: EventQueue, event: ContractEvent) {
+    this.queue.push(event);
+    verbose(
+      'Event added to the queue:',
+      event?.signature || ContractEventsSignatures.UnknownEvent,
```

Comment:
Changes in this file are just from prettier & correcting the "Uknown" event typo